dev/example-config.json

```diff
@@ -82,10 +82,10 @@
     "cacert": "config/myCA.crt",
     "mtls_client_cert": "config/mtls_auth.crt",
     "mtls_client_key": "config/mtls_auth.key",
-    "url": "https://vault:8300",
-    "namespace": "",
+    "url": "https://127.0.0.1:8300",
+    "namespace": "tenant",
     "auth_jwt_path": "jwt",
     "default_role": "distributey",
-    "transit_path": "transit"
+    "transit_path": "transit-monitoring"
```

Comment:
Why do you suggest to set the global vault block to the `transit-monitoring` path? Namespace is set to `tenant` here, but the monitoring tenant's namespace is set to root.